Doc/tools/templates/download.html

```diff
@@ -13,7 +13,7 @@ <h1>Download Python {{ release }} Documentation</h1>
 
 <p>To download an archive containing all the documents for this version of
 Python in one of various formats, follow one of links in this table. The numbers
-in the table are the size of the download files in megabytes.</p>
+in the table are the size of the download files in megabytes (2<sup>20</sup> bytes).</p>
```

Comment:
```
1 megabyte = 10**6 bytes
1 mebibyte = 2**20 bytes
```

Reply:
@vstinner I followed your comment on https://github.com/python/cpython/pull/9639#discussion_r231696589

Perhaps your "megabytes" was a typo? I think mebibyte is the correct thing to say, by the way.

Reply:
@serhiy-storchaka I changed the unit based on your comment.